src/services/EventsExpressService.js

```diff
@@ -32,14 +32,14 @@ export default class EventsExpressService {
   };
 
   setResource = async (Url, data) => {
-    const call = (Url, data) =>
-      fetch(this.baseUrl + Url, {
+    const call = (url, Data) =>
```

Comment:
Why you write Data from big letter? If you need another parameters from function setResource, you will write callUrl and callData.